cmd/pj-rehearse/main.go

```diff
@@ -141,3 +142,13 @@ func main() {
 		gracefulExit(o.noFail)
 	}
 }
+
+func createMasterWorktree(cadidatePath, baseRSA string) error {
+	var out bytes.Buffer
+	cmd := exec.Command("git", "-C", cadidatePath, "worktree", "add", "master", baseRSA)
+	cmd.Stdout = &out
```

Comment:
What is this used for? Could just use `CombinedOutput()` to get `stdout` and `stderr` and print on failures.